common/djangoapps/student/migrations/0043_remove_userprofile_allow_certificate.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.20 on 2021-05-06 04:56
+
+from django.db import migrations
```

Comment:
why this migration is added?

Reply:
It seems this migration for temporarily left out in the previous PR: https://github.com/edx/edx-platform/pull/27424
So it was added when I generated migrations. 

I'll remove it since it's unrelated.

Reply:
@xitij2000, the migration is still here. Dropping a column consists of [two steps](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations#EverythingAboutDatabaseMigrations-Howtodropacolumn), so we shouldn't introduce it here.